joomla/joomla-cms
# [4.0] Implement char counter for input field, too

### Summary of Changes
Enabled character counter for text fields


### Testing Instructions
1. Apply patch
2. NPM i
3. Open components/com_users/forms/registration.xml
4. Add the following attributes to the "name" field:
`
maxlength="30"
charcounter="true"`
5. Go to the frontend registration and check the name field


### Actual result BEFORE applying this Pull Request
Nothing different


### Expected result AFTER applying this Pull Request
![grafik](https://user-images.githubusercontent.com/1229869/134845459-0960eb77-1388-45b4-871f-882fa6c7eeca.png)

![grafik](https://user-images.githubusercontent.com/1229869/134845491-cfd6e6a2-2129-43ec-945f-a28739279203.png)
